lib/config/types.ts

```diff
@@ -561,3 +562,15 @@ export interface ValidationResult {
   errors: ValidationMessage[];
   warnings: ValidationMessage[];
 }
+
+/**
+ * Experimental Flags can have the following values:
+ * 1. integer                // dockerMaxPages=10
+ * 2. string                 // s3Endpoint=some-url
+ * 3. json-strings       // mergeConfidenceSupportedDatasources=[docker,azure]
+ * 4. boolean               //  dockerHugTags=true
```

Comment:
I think boolean can be flat, no need for =true

Reply:
https://github.com/renovatebot/renovate/pull/28187#discussion_r1557403395
@viceice said we might reverse the flag values, in that case we will need it.

Reply:
No, we should always make Boolean ones opt-in true

Reply:
No need to ever set =false